README.md

```diff
@@ -118,7 +118,8 @@ POSTing of entries requires sending the following payload object:
   }
 
   tags: optional array of strings
-  issues: optional array of strings, must match values from [GET /issues?format=json]
+  issue: optional string, must match value from [GET /issues?format=json]
+  help_type: optional string, must match value from [GET /helptypes?format=json]
```

Comment:
I went with `help_type` (instead of the `helptype`) since `page_size` is in snake_case already